assets/vimrc

```diff
@@ -0,0 +1,86 @@
+if v:lang =~ "utf8$" || v:lang =~ "UTF-8$"
```

Comment:
I don't think we need to install a vim configuration into our Docker images. In general we shouldn't be editing things inside the image.

Reply:
I find it useful for debugging things within the container.  I will not have in the production container - however nothing seems to be dependent on vim, so it is easily removed.

Reply:
Ok, I have removed vi from the "production" container and moved it to a "dev" container based off of the production container.

the build configs/image streams have yaml for them.